### PR TITLE
Improved looktype.lua

### PR DESCRIPTION
**Pull request by LucasCPrazeres**
- Reduction in the size of **if**.
- parameter check if it's number

Sorry for my English. :(

### CODE REVIEW

**Comment by ranisalt** [comment]
Why don't you add all ids to the list?

**Comment by WibbenZ** [comment]
I would space them out on new lines, longer to read but so much easier to find and add stuff :p

**Comment by Tarliton** [comment]
what about this?

```lua
local minLookType = 2
local maxLookType = 699
local invalidLookTypes = {161, 162, 163, 164, 165, 166, 167, 168, 
                          169, 170, 171, 172, 173, 174, 175, 176, 
                          177, 178, 179, 180, 181, 182, 183, 184, 
                          185, 186, 187, 188, 189, 190, 191, 474, 
                          475, 476, 477, 478, 479, 480, 481, 482, 
                          483, 484, 485, 638, 639, 640, 641, 642, 
                          643, 645, 646, 652, 653, 654, 655, 656, 
                          657, 658, 659, 660, 661, 662, 663, 135, 
                          411, 415, 424, 439, 440, 468, 469, 518, 
                          519, 520, 524, 525, 536, 543, 549, 576, 
                          581, 582, 597, 616, 623, 635}


function onSay(player, words, param)
    if not player:getGroup():getAccess() then
        return true
    end

    local lookType = tonumber(param)

    if not lookType then
        player:sendCancelMessage("Param incorrect.")
        return false 
    end

    if lookType >= minLookType and lookType <= maxLookType and not isInArray(invalidLookTypes, lookType) then
        local playerOutfit = player:getOutfit()
        playerOutfit.lookType = lookType
        player:setOutfit(playerOutfit)
    else
        player:sendCancelMessage("A look type with that id does not exist.")
    end

    return false
end
```

**Comment by LucasCPrazeres** [comment]
@ranisalt Being honest, I was a little laziness.

@WibbenZ One idea is to facilitate the reading of the programmer.

@Tarliton tanks

**Comment by ranisalt** [comment]
Your indentation is entirely wrong. TFS code style mandates tab as indentation level, not spaces. Also, break a line just after opening braces and before closing braces, and indent the content one leve.

File: data/talkactions/scripts/looktype.lua [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
-	end
-	return false
+    if not player:getGroup():getAccess() then
+        return true
+    end
+
+    local lookType = tonumber(param)
+
+    if not lookType then
+        player:sendCancelMessage("Param incorrect.")
```
Comment (ranisalt) [inline data/talkactions/scripts/looktype.lua]: Please revert the message to what was there before.

File: data/talkactions/scripts/looktype.lua [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+        return false 
+    end
+
+    if lookType >= minLookType and lookType <= maxLookType and not isInArray(blockLookType, lookType) then
```
Comment (ranisalt) [inline data/talkactions/scripts/looktype.lua]: You definitely don't need this min and max look type. Everything inside the array is between the boundaries, so the check is redundant.

**Comment by ranisalt** [comment]
As the dev didn't answer I pushed a fixed version instead.

**Pull request closed without merging**